.circleci/config.yml

```
@@ -47,16 +47,24 @@ jobs:
           root: .
           paths:
             - .
-  lint_backend_and_frontend:
+  lint_backend:
     executor: docker-executor
     steps:
       - attach_workspace:
           at: .
       - run:
-          name: Lint backend and frontend
-          command: yarn run lint:ci && cd frontend && yarn run lint:ci
+          name: Lint backend
+          command: yarn run lint:ci
```

```suggestion
          command: yarn run lint:ci
      - run:
          name: Lint frontend
          command: yarn --cwd frontend run lint:ci
```
Add a step to run the command for linting the frontend.